tests/testthat/test-actual-distributions.R

```diff
@@ -0,0 +1,45 @@
+library(hedgehog)
+freeze_eval <- names(.GlobalEnv)
```

Comment:
As you are not using hedgehog in this file you can remove lines 1,2 and 45 :)

Reply:
Done.